doc/news/_preparation_next_release.md

```diff
@@ -53,7 +53,26 @@ The following section lists news about the [plugins](https://www.libelektra.org/
 - <<TODO>>
 - <<TODO>>
 
-### Blacklist
+### Length
+
+- Implement a plugin that validates that a string length is less or equal to given number. _(Philipp Oppel)_
+  _(Michael Tucek)_
+
+Special thanks to _(Klemens Böswirth)_, _(Mihael Pranjić)_ and _(Robert Sowula)_ for helping with the problem analysis!
+
+### Dbus
+
+- Internal changes to ensure compatibility with the new `elektraNotificationContract`. _(Klemens Böswirth)_
+
+### Dbusrecv
+
+- Internal changes to ensure compatibility with the new `elektraNotificationContract`. _(Klemens Böswirth)_
+
+### Xerces
+
+- Store length of an array in metakey array according to [array decision](../decisions/array.md). _(Robert Sowula)_
+
+### YAML Smith
```

Comment:
This whole part is from another release. I will rebase and fix the release notes.

```suggestion

```